Completes issue #412 - Add a show password button to the Login and Si…

amygarciaf opened the pull request:
Completes issue #412 - Add a show password button to the Login and Sign up Screens

amygarciaf committed:
Completes issue #412 - Add a show password button to the Login and Sign up screens

julianlopez6850 reviewed (commented):
The show password button works and looks great. 

However, on the `Create Password` page during Sign Up, I noticed that the button only shows up on the `Password` input, but it doesn't show up on the `Repeat Password` input. I wasn't sure if you excluded it from the Repeat Password portion on purpose, or if this is a mistake (I think it would make sense for it to show up on both, but it doesn't really matter). Let me know so I can review again and approve.

amygarciaf committed:
Update to PR #436 - Adds Show Password button to the Repeat Password Input

Adds Show Password button to the Repeat Password Input (Issue #412)

amygarciaf commented:
Thank you. The button was excluded on purpose, I was undecided about adding or excluding it from Repeat Password portion. I think you're right, it should be more useful for it to show up on both inputs! I just modified the **SignUpPassword.jsx** file and updated the PR.

julianlopez6850 reviewed (approved):
looks great!
